changeset_test.go

```diff
@@ -206,3 +215,110 @@ func TestChangesetApply_constraint(t *testing.T) {
 		UpdatedAt: now,
 	}, user)
 }
+
+func TestChangesetApply_MutablePK(t *testing.T) {
```

Comment:
for the name case, the convention in this repo is to start with lower case
```suggestion
func TestChangesetApply_mutablePK(t *testing.T) {
```

maybe mutablePK can be changed as well since we are not using that variable anymore

Reply:
Tests are updated